### ISSUE TITLE
Spotify Login Form does alway stay on top of all other apps

### ISSUE THREAD

**Issue by maximilianmeier** [davicorreiajr/spotify-now-playing#7]
## What are you reporting?

* [x] Bug or some problem;

## In which context are you running the app?
macOS Mojave 10.14.1
App Version: 0.2.0

## Description
When you first start the app, you will be prompt with the Spotify login form. this login form will always stay on top of all other windows. so you can check other apps at that time. Also, CMD+Q does not work in this window. you have to close the app over the menu bar.

**Comment by H8mut** [comment]
Have the same issue and don't have my spotify log in data accessable right now.
It should be at least closable ;-)  So a cancel button would be great.

**Comment by Ceany** [comment]
Thanks for the quick response.

I think a cancel button or the standard `CMD+Q` option should available. Additionally, I don't think it is necessary for the window to stay at the top all the time. You should be able to switch back anytime with `CMD+Tab`.

**Comment by Ceany** [comment]
I got logged out by Spotify. I logged in again. This windows also stayed always at the top of all other applications until I entered my credentials. but the weird part is, that now when I open the app I just get the Spotify window that I am logged in. The app actually does not work now. See screenshot:

<img width="806" alt="spotify-playing-bug" src="https://user-images.githubusercontent.com/10998002/49344113-a5e94400-f673-11e8-94ec-b0370ac91f70.png">
